services/core-web/src/components/Forms/permits/SitePropertiesForm.js

```diff
@@ -0,0 +1,195 @@
+import React, { Component } from "react";
+import { PropTypes } from "prop-types";
+import { Row, Col, Popconfirm, Button } from "antd";
+import { Field, formValueSelector, FormSection, reduxForm, Form } from "redux-form";
+import { connect } from "react-redux";
+import { compose } from "redux";
+import RenderMultiSelect from "@/components/common/RenderMultiSelect";
+import RenderSelect from "@/components/common/RenderSelect";
+import CustomPropTypes from "@/customPropTypes";
+import { requiredList, maxLength, required } from "@common/utils/Validate";
+import {
+  getConditionalDisturbanceOptionsHash,
+  getConditionalCommodityOptions,
+  getMineTenureTypeDropdownOptions,
+  getExemptionFeeSatusDropDownOptions,
+} from "@common/selectors/staticContentSelectors";
+import { determineInspectionFeeStatus } from "@common/utils/helpers";
+import * as FORM from "@/constants/forms";
+import RenderAutoSizeField from "@/components/common/RenderAutoSizeField";
+/**
+ * @constant SitePropertiesForm renders edit/view for the NoW Application review step
+ */
+
+const propTypes = {
+  permit: CustomPropTypes.permit.isRequired,
+  mineTenureTypes: PropTypes.objectOf(CustomPropTypes.options).isRequired,
+  conditionalCommodityOptions: PropTypes.objectOf(CustomPropTypes.options).isRequired,
+  conditionalDisturbanceOptions: PropTypes.objectOf(CustomPropTypes.options).isRequired,
+  site_properties: PropTypes.objectOf(
+    PropTypes.oneOfType([PropTypes.string, PropTypes.arrayOf(PropTypes.string)])
+  ).isRequired,
+  exemptionFeeSatusDropDownOptions: PropTypes.objectOf(CustomPropTypes.options).isRequired,
+  submitting: PropTypes.bool.isRequired,
+  closeModal: PropTypes.func.isRequired,
+  handleSubmit: PropTypes.func.isRequired,
+  change: PropTypes.func.isRequired,
+};
+
+const mapApplicationTypeToTenureType = (permitPrefix) =>
```

Comment:
This has been defined slightly differently in a few files?

Reply:
The mapping is slightly different depending if it occurs on the permit or the NoW flow (which has been commented out) 

In this sile we already know what the permitPrefix is, so there is no need to supply a null option. The AddPermitForm has a null option, and the NoW flow separates the two Q types vs grouping them together as the permit does.